src/components/Breadcrumb/index.tsx

```diff
@@ -15,26 +16,37 @@ export const BreadCrumb: React.FC<Props> = ({
   isTask = false
 }) => {
   return (
-    <Breadcrumb>
-      <Breadcrumb.Item>
+    <Breadcrumb style={{ marginBottom: "25px" }}>
+      <StyledBreadcrumItem link={true}>
         <span id="bc-my-patches">
           <Link to={paths.myPatches}>My Patches</Link>
         </span>
-      </Breadcrumb.Item>
-      <Breadcrumb.Item>
+      </StyledBreadcrumItem>
+      <StyledBreadcrumItem link={isTask}>
         <span id="bc-patch">
```

Comment:
I also get this warning:
`Warning: [antd: Breadcrumb] Only accepts Breadcrumb.Item and Breadcrumb.Separator as it's children`